analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Sonar/Analyzers/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -153,7 +151,7 @@ private ProgramState VisitInvocationExpression(InvocationExpressionSyntax instru
 
                         if (disposableSymbol is IMethodSymbol
                             // Special case - if the parameter symbol is "this" then resolve it to the containing type
-                            || (disposableSymbol is IParameterSymbol parameter && parameter.IsThis))
+                            || disposableSymbol is IParameterSymbol parameter && parameter.IsThis)
```

Comment:
I think it would make sense to create a .editorconfig for our sln and activate this rule:
![image](https://user-images.githubusercontent.com/103252490/180163196-f34f627e-78c7-46fc-8989-5ce3c66595b9.png)
Otherwise, the clean-up tool in VS removes the parenthesis.

Reply:
I just found it. We disable it:
![image](https://user-images.githubusercontent.com/103252490/180165928-75ad1c29-f7c4-42a3-881f-15d409b3de80.png)
I still don't get why the fixer was applied by the clean-up tool because I do not include it:
![image](https://user-images.githubusercontent.com/103252490/180166382-7cc5ba2b-dafd-45d4-9d4e-ba13925efb6f.png)

Do you want me to revert any changes to parenthesis?